src/Grids/vertically_stretched_cartesian_grid.jl

```diff
@@ -90,13 +90,21 @@ function VerticallyStretchedCartesianGrid(FT=Float64, arch=CPU();
      xC = OffsetArray(xC,  -Hx)
      yC = OffsetArray(yC,  -Hy)
      zC = OffsetArray(zC,  -Hz)
-    ΔzC = OffsetArray(ΔzC, 1 - Hz)
+    ΔzC = OffsetArray(ΔzC, -Hz)
 
      xF = OffsetArray(xF,  -Hx)
      yF = OffsetArray(yF,  -Hy)
      zF = OffsetArray(zF,  -Hz)
     ΔzF = OffsetArray(ΔzF, -Hz)
 
+    # Needed for pressure solver solution to be divergence-free.
+    # Will figure out why later...
+    ΔzC[Nz] = ΔzC[Nz-1]
+
+    # Seems needed to avoid out-of-bounds error in viscous dissipation
+    # operator wanting to access ΔzC[Nz+2].
+    ΔzC = OffsetArray(cat(ΔzC[0], ΔzC..., ΔzC[Nz], dims=1), -Hz-1)
```

Comment:
Trying to figure out if this makes sense...

Reply:
Yeah I'm not sure either but everything passed when I added it so I just need to understand why it's needed. A similar issue cropped up for the `CompressibleModel` where we were sure we just needed halos of size 1 but the viscous dissipation operators wanted halos of at least size 2 because ρu and ρu / ρ create more interpolation.

Reply:
Do `closure=nothing` tests pass with ordinary halos?

Could also be an off-by-one mistake somewhere ?